Client/Frontend/Browser/BrowserViewController/BrowserViewController+WebViewDelegates.swift

```diff
@@ -385,16 +385,6 @@ extension BrowserViewController: WKNavigationDelegate {
         //            return
         //        }
 
-        // Second special case are a set of URLs that look like regular http links, but should be handed over to iOS
-        // instead of being loaded in the webview. Note that there is no point in calling canOpenURL() here, because
-        // iOS will always say yes.
-
-        if isAppleMapsURL(url) {
```

Comment:
I think there is a better solution to this than removing this code. It goes something like this:
Inside `extension BrowserViewController: WKNavigationDelegate` add a couple of static vars:
   ```
    static var lastAppleMapsURL: URL?
    static var lastAppleMapsURLTime: Date?
```

Use something like this in place of the original appleMaps handling:
```
        if isAppleMapsURL(url) {
            decisionHandler(.cancel)
            if url == Self.lastAppleMapsURL,
               Self.lastAppleMapsURLTime!.timeIntervalSinceNow <= 0.1 {
                // Looks like Maps isn't around to handle the request below and it got routed back to us. Ignore it
            } else {
                UIApplication.shared.open(url, options: [:])
                Self.lastAppleMapsURL = url
                Self.lastAppleMapsURLTime = Date()
            }
            return
        }

```
(Maybe the 0.1 second time interval needs tweaking)

Unfortunately I can't test that here locally as I don't have a provisioning profile that allows the default browser entitlement

Reply:
@gutley if you pull `main`, we no longer require that provisioning profile for local builds.